solar-system.rb

```diff
@@ -0,0 +1,101 @@
+class SolarSystem
+  # Helper method w/ @planets instance variable representing the array of planet names (set below)
+  attr_accessor :planet_list
+
+  # Constructor (so when you create a new object that is an instance of this class using .new, you will need to include an argument for the list of planet names
+  def initialize(planet_list)
+    @planet_list = planet_list
+  end
+
+  # Add planet to the list
+  def add_planet(planet)
+    @planet_list << planet
+    return @planet_list
+  end
+
+  # Method to show the full list of planets
+  def list_planets
+    list = ""
+    num = 1
+    @planet_list.each do |planet|
+      list += "#{num}. #{planet.planet_name} \n"
+      num += 1
+    end
+    return list
+  end
+end
+
+class Planet
+  # Helper method w/ instance variables representing a given planet's attributes
+  attr_accessor :planet_name, :distance_from_the_sun, :diameter, :year_length, :number_of_moons, :random_fact
+
+  # Constructor
+  def initialize(planet_name, distance_from_the_sun, diameter, year_length, number_of_moons, random_fact)
+    @planet_name = planet_name
```

Comment:
Since it's an instance variable on `Planet`, calling it `planet_name` is a little redundant.